ui/app/components/ui/code-editor.tsx

```diff
@@ -153,7 +154,8 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
     autoDetectLanguage ? detectLanguage(value) : allowedLanguages[0],
   );
 
-  const [wordWrap, setWordWrap] = useState(
+  const [wordWrap, setWordWrap] = useLocalStorage(
+    "word-wrap",
```

Comment:
Since `DEFAULT_WORD_WRAP_LANGUAGES` suggests different languages should have different defaults, would we want the local storage value to be global across all languages, or have it set per-language?

I'm leaning towards it being "global", but I'm not sure I have enough context of how this is being used across the dashboard to make a final decision. Thoughts?

Reply:
I think let's start with global, don't have an intuition for what would be different from global right now.